test/src/test/java/test/org/dst/benchmark/core/DSTBenchmark.java

```diff
@@ -0,0 +1,77 @@
+package test.org.dst.benchmark.core;
+
+import org.dst.client.DefaultDstClient;
+import org.dst.client.DstClient;
+import test.org.dst.supplier.TestUtil;
+import java.util.ArrayList;
+import java.util.List;
+
+import static test.org.dst.benchmark.DstBenchmarkTest.benchmarkTest;
+
+public class DSTBenchmark {
+  public static final String serverAddress = "list://127.0.0.1:8082";
+
+  private int threadNum = 1;
+
+  private Runnable myTest = null;
+
+  public void setTestModule(Runnable myTest) {
+    this.myTest = myTest;
+  }
+
+  public void setThreadNum(int threadNum) {
+    this.threadNum = threadNum;
+  }
+
+  public void dstRun() {
```

Comment:
It's too confused between the name `dstRun` and `run`.
You should refine it. It means you should give them reasonable names.